docs/source/upgrade.rst

```diff
@@ -8,6 +8,26 @@ This is a list of changes introduced in each release that users should be aware
 Most changes are carefully designed not to break existing code; however changes that may possibly break them are highlighted with a box.
 
 
+Chainer v5
+==========
+
+Changed Behavior of ``Optimizer.new_epoch``
+-------------------------------------------
+
+This change only affects to a minor number of users (who call ``new_epoch()`` method of Optimizers while using a trainer, or who implement their own Updater class).
```

Comment:
* "This change should affect only a minority of users"
* ... ```:meth:`Optimizer.new_epoch` ``` method manually
* ```:class:`Updater` ``` class